src/main/java/sx/blah/discord/util/PermissionUtils.java

```diff
@@ -458,4 +478,67 @@ private static boolean hasHigherRoles(List<IRole> roles1, List<IRole> roles2) {
 		set.addAll(Arrays.asList(array));
 		return set;
 	}
+
+	/**
+	 * Gets the permissions a user has in some permissions object, taking into account user and role overrides.
+	 *
+	 * @param user The user to get permissions for.
+	 * @param guild The guild the user is in.
+	 * @param userOverrides The internal user overrides cache.
+	 * @param roleOverrides The internal role overrides cache.
+	 * @return The permissions the user has in the permissions object.
+	 */
+	public static EnumSet<Permissions> getModifiedPermissions(IUser user, IGuild guild,
+															  Cache<PermissionOverride> userOverrides,
```

Comment:
Strange formatting?

Reply:
Blame tabs, it looks fine in the editor.